tests/components/ResultsFilter/ResultsFilter.spec.ts

```diff
@@ -42,6 +43,23 @@ describe('ResultsFilter', () => {
         expect(filter.cmp.isSelected()).toBeFalsy();
     });
 
+    it('should trigger events on toggling', done => {
+        let cpt = 0;
+        Coveo.$$(filter.env.root).on(ResultsFilterEvents.Click, (evt: Event, args: IResultsFilterEventArgs) => {
+            expect(evt.type).toBe(ResultsFilterEvents.Click);
+            if (args.checked) {
```

Comment:
What's the point of this if-else ?

Reply:
Make sure when I toggle two times the event has both values

Reply:
Yeah but... You have exactly the same statements in the if and else